mmhuman3d/utils/img_utils.py

```diff
@@ -0,0 +1,140 @@
+"""This file contains functions that are used to perform data augmentation."""
```

Comment:
Shall we re-organize this file into [data pipeline](https://github.com/open-mmlab/mmhuman3d/blob/main/mmhuman3d/data/datasets/pipelines/)?

- For common ones, we can re-use the implemented version, e.g., replacing `flip_kp` with [`_flip_keypoints`](https://github.com/open-mmlab/mmhuman3d/blob/main/mmhuman3d/data/datasets/pipelines/transforms.py#L253)
- For speficial functions related to PyMAF-X only, we can write them in a separate file like [hybrik_transforms](https://github.com/open-mmlab/mmhuman3d/blob/main/mmhuman3d/data/datasets/pipelines/hybrik_transforms.py)